libs/bookings/src/lib/booking.utilities.ts

```diff
@@ -15,7 +15,7 @@ export function generateBookingForm(booking: Booking = new Booking()) {
         title: new FormControl(booking.title),
         description: new FormControl(booking.description),
         booking_asset: new FormControl(null),
-        asset_id: new FormControl(booking.asset_id, [Validators.required]),
+        asset_id: new FormControl(booking.asset_id),
```

Comment:
Can you undo this as this should be a required field.
If you are doing full validity checks it should be at the last step in the flow.

Reply:
Had previously added this but found that it's made the old flow stop working.

Reply:
Undo changes as discussed